### PR TITLE
[DOC] Expired slack link under "Where to ask questions"

### PR DESCRIPTION
**Pull request by topher-lo**
Fixes expired link to Slack under the "Where to ask questions" section of the README.

Related to #3014

### CODE REVIEW

**Comment by aiwalter** [comment]
where is this "Where to ask questions" section?

**Comment by aiwalter** [comment]
are you on the "stable" version of the website?

File: README.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ Questions and feedback are extremely welcome! Please understand that we won't be
 [github discussions]: https://github.com/alan-turing-institute/sktime/discussions
 [stack overflow]: https://stackoverflow.com/questions/tagged/sktime
 [discord]: https://discord.com/invite/gqSab2K
-[slack]: https://join.slack.com/t/sktime-group/shared_invite/zt-62i7aejn-vXc3nOWF26S_P3VXFPWisQ
+[slack]: https://join.slack.com/t/sktime-group/shared_invite/zt-1cghagwee-sqLJ~eHWGYgzWbqUX937ig
```
Comment (aiwalter) [inline README.md:61]: this is already the current link. I am wondering why git shows this is a change

Comment (topher-lo) [reply README.md:61]: Hi! I took a screenshot of the relevant section of the README at posted it in the associated GitHub issue. Can confirm it is the stable version. There are two slack links on the README. One which was updated previously. And this one which was not noticed

Comment (topher-lo) [reply README.md:61]: The screenshot is in #3014 and #3017 seems to only fix 1 of the 2 links in the README

**Review by aiwalter — APPROVED** [review]

**Pull request merged**